projects/ngrx.io/content/guide/store/index.md

```diff
@@ -48,6 +49,8 @@ The following tutorial shows you how to manage the state of a counter, and how t
 
 6.  Create a new _Component_ named `my-counter` in the `app` folder. Inject the `Store` service into your component to dispatch the counter actions, and use the `select` operator to _select_ data from the state.
 
+Because we are using selectors to access data from the `Store` it is not necessary to provide as generic type to the `Store`. Previously, you would need to provide a type such as `Store<AppState>`, however, this is no longer necessary as the default `Store` provides a default generic type.
```

Comment:
The "Getting started" guide is using `select('count')`, so it would still need the typed Store.

I think the section wrt using generic for Store should live at https://ngrx.io/guide/store/selectors. Moreover, it should have a header, so it can be easily referred to from outside. (shared link)

Reply:
Sounds good. I’ll update this tomorrow.